opensearch-operator/pkg/helpers/helpers_suite_test.go

```diff
@@ -3,11 +3,78 @@ package helpers
 import (
 	"testing"
 
+	"github.com/Opster/opensearch-k8s-operator/opensearch-operator/mocks/github.com/Opster/opensearch-k8s-operator/opensearch-operator/pkg/reconcilers/k8s"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestHelpers(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Helpers Suite")
 }
+
+var _ = Describe("Helpers", func() {
```

Comment:
This is not the correct file for the tests, they should go in a file `helpers_test.go`, the `helpers_suite_test.go` only contains the init/start code for the test of the entire package.